usaspending_api/agency/v2/views/award_count.py

```diff
@@ -129,7 +129,8 @@ def _get_award_counts_response(self, filters: List[dict], filter_options: dict)
         Returns:
             List[dict]: A dictionary containing award types and the number of their awards.
         """
-        filter_query = QueryWithFilters.generate_awards_elasticsearch_query(filters, **filter_options)
+        query_with_filters = QueryWithFilters(_QueryType.AWARDS)
```

Comment:
This comment is unrelated to this line specifically, but applies with its use of `_QueryType`.

If it is not too much of a pain, would you mind updating `_QueryType` to instead be `QueryType`? In our codebase we have a few instances where the `_` is prepended to signify a "private" method or property, but then it gets used in a lot of places outside of the file where it is defined. It would be good to start cleaning this up so that we are more intentional and confirming to Python standards.